apps/lockfile-explorer/src/utils/init.ts

```diff
@@ -18,60 +18,60 @@ export const init = (options: {
   subspaceName: string;
 }): IAppState => {
   const { lockfileExplorerProjectRoot, appVersion, debugMode, subspaceName } = options;
-  const currDir = process.cwd();
+  const currentWorkingDirectory = process.cwd();
 
   let appState: IAppState | undefined;
-  let currExploredDir = Path.convertToSlashes(currDir);
-  while (currExploredDir.includes('/')) {
+  let currentFolder = Path.convertToSlashes(currentWorkingDirectory);
+  while (currentFolder.includes('/')) {
     // Look for a rush.json [rush project] or pnpm-lock.yaml file [regular pnpm workspace]
-    const rushJsonPath: string = path.resolve(currExploredDir, 'rush.json');
-    const pnpmLockPath: string = path.resolve(currExploredDir, 'pnpm-lock.yaml');
+    const rushJsonPath: string = path.resolve(currentFolder, 'rush.json');
```

Comment:
Are these ever going to be non-absolute paths? If they're always absolute, just concatenate them with a `/`

Reply:
BTW `path.resolve` is more expensive than `path.join` (which is also much more expensive than `` `${path1}/${path2}` ``